From d23fcd7a93e2a961b81c413392d41b3dbc7b4c70 Mon Sep 17 00:00:00 2001
From: Joshua Downer <joshua.downer@gmail.com>
Date: Tue, 4 Aug 2020 14:52:21 -0400
Subject: [PATCH 1/2] configure.ac: use PKG_CHECK_MODULES

Replace AC_CHECK_{LIB,HEADER} with PKG_CHECK_MODULES and augment CFLAGS
and LIBS with LIBUSB_{CFLAGS,LIBS} information.
---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index ab96f44..efc1632 100644
--- a/configure.ac
+++ b/configure.ac
@@ -60,9 +60,9 @@ case $host in
 esac
 
 # Checks for libraries.
-AC_CHECK_LIB([usb-1.0], [libusb_init], [], [libul_lib_error libusb-1.0])
-
-AC_CHECK_HEADERS([libusb-1.0/libusb.h], [], [libul_lib_error libusb-1.0])
+PKG_CHECK_MODULES([LIBUSB], [libusb-1.0], [], [libul_lib_error libusb-1.0])
+CFLAGS="${CFLAGS} ${LIBUSB_CFLAGS}"
+LIBS="${LIBS} ${LIBUSB_LIBS}"
 
 # OS info for Automake
 AM_CONDITIONAL(OS_LINUX, test "x$os" = xlinux)
-- 
2.17.1

